assets/dev/js/editor/elements/views/base.js

```diff
@@ -470,7 +470,34 @@ BaseElementView = BaseContainer.extend( {
 
 	renderStyles( settings ) {
 		if ( ! settings ) {
-			settings = this.getEditModel().get( 'settings' );
+			if ( elementorCommonConfig.isTesting ) {
+				settings = this.getEditModel().get( 'settings' );
+
+				return this.renderStyles( settings );
+			}
+
+			const request = $e.data.get( 'document/elements', {
+				document_id: elementor.documents.getCurrent().id,
+				element_id: this.getEditModel().id,
+			}, {
+				cache: true,
+				cacheOnly: true, // Cannot cache, Create section, columns.
```

Comment:
Try work with `targetContainer.view.allowRender = false;` on create section columns.

Reply:
not possible.